docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
         ports:
             - 3000:3000
         healthcheck:
-            test: ['CMD-SHELL', 'curl -s http://localhost:3000/']
+            test: ['CMD-SHELL', 'curl -s http://localhost:3000/api/health']
```

Comment:
@chrisbendel I updated the link here as `localhost:3000/` will be blocked by the `basic-auth` and might return `403` exception.